.github/workflows/verify_consumer_pacts.yaml

```diff
@@ -0,0 +1,211 @@
+name: Verify consumer pacts
+# The purpose of this workflow is to verify ANY consumer contract(s) dependent on workspacemanager provider using Pact
+# framework.
+#
+# The workflow meets the criteria of Pact Broker *Platinum* as described in https://docs.pact.io/pact_nirvana/step_6.
+# The can-i-deploy job has been added to this workflow to gate the merge of PRs into develop branch.
+#
+# This workflow is triggered when
+#
+# 1. Consumer makes a change that results in a new pact published to Pact Broker (will verify ONLY the changed pact and publish the verification results back to the broker)
+# 2. Provider makes a change (runs verification tests against ALL DEPLOYED consumer pact versions and publishes corresponding verification results)
```

Comment:
> ALL DEPLOYED consumer pact versions

doesn't this require some configuration/integration with beehive or sherlock? Is this already set up?

Reply:
Honestly, I'm not sure...this file is the result of a blatant copypaste.  I did check in https://beehive.dsp-devops.broadinstitute.org/charts/workspacemanager/contract-testing and confirmed that for `workspacemanager` contract testing is enabled:
<img width="471" alt="image" src="https://github.com/DataBiosphere/terra-workspace-manager/assets/111856/c1f22c2e-59f5-4095-a59d-ac8813f330c3">

Reply:
I'll get to exercise some of the provider side again after this merges by integrating another couple pact verifications...hopefully I'll have more of a clue after that process.